<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pagerid</class>
 <widget class="QDialog" name="Pagerid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>147</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pager IDs</string>
  </property>
  <widget class="QPushButton" name="pbDone">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>306</y>
     <width>67</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Done</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwPagerIDs">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>143</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>306</y>
     <width>67</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
